src/utils.coffee

```diff
@@ -18,7 +18,7 @@ device = require './device'
 exports.supervisorVersion = require('./lib/supervisor-version')
 
 configJson = require('/boot/config.json')
-if Boolean(configJson.supervisorOfflineMode)
+if Boolean(config.apiEndpoint) and !Boolean(configJson.supervisorOfflineMode)
```

Comment:
Would it make sense to move this `and` into the config.coffee file so we don't have to do the double check in every case and can instead use `if config.offlineMode`?